vignettes/EIRprevmatch.Rmd

```diff
@@ -89,85 +173,209 @@ prev <- lapply(
+# for plotting
+malSim_fit <- cbind(malSim_fit, data.frame(init_EIR = c(0 ,seq(0.1, 50, 0.1))))
+
+```
+
+## Visualisation
 
-prevmatch <- cbind.data.frame(eir, prev)
+Let's visually compare our the `malariaEquilibrium` and `malariasimulation` methods for matching EIR to *Pf*PR~2-10~ values. In the section below we open a blank plot, plot the initial EIR and resulting *Pf*PR~2-10~ points generated using `malariasimulation` runs, overlay the line of best fit (orange line). Also overlayed is a line mapping EIR and *Pf*PR~2-10~ values calculated using `malariaEquilibrium` (blue line).
 
-# calculate best fit line through malariasimulation data
-fit <- predict(gam(prev~s(init_EIR, k=5), data=EIR_prev), 
-               newdata = data.frame(init_EIR=c(0,seq(0.1,50,0.1))),  type="response")
-fit <- cbind(fit, data.frame(init_EIR=c(0,seq(0.1,50,0.1))))
+```{r,  fig.width = 6.8, fig.height = 4}
 
-# plot
-plot(x=1, type = "n", 
+# Define a colour palette for plotting:
+plot_cols <- c("#E69F00", "#56B4E9", "#009E73", "#CC79A7","#F0E442", "#0072B2", "#D55E00")
+
+# Establish a plotting window:
+plot(x = 1, type = "n", 
      frame = F, 
-     xlab = "initial EIR", 
-     ylab = expression(paste(italic(Pf),"PR"[2-10])),
-     xlim = c(0,50), 
-     ylim = c(0,.7))
+     xlab = "Initial EIR", ylab = expression(paste(italic(Pf),"PR"[2-10])),
+     xlim = c(0,50), ylim = c(0, 1),
+     xaxs = "i", yaxs = "i")
 
-points(x=EIR_prev$init_EIR, 
-       y=EIR_prev$prev,
+# Overlay the initial EIR and corresponding PfPR2-10 points from malariasimulation
+points(x = malSim_P2E$init_EIR, 
+       y = malSim_P2E$prev,
        pch = 19,
        col = 'black')
 
-lines(x=fit$init_EIR, 
-      y=fit$fit, 
-      col = "red", 
+# Overlay the malariasimulation line of best fit:
+lines(x = malSim_fit$init_EIR, 
+      y = malSim_fit$malSim_fit, 
+      col = plot_cols[1],
+      lwd = 2,
       type = "l", 
       lty = 1)
 
-lines(x=prevmatch$eir, 
-      y=prevmatch$prev, 
-      col = "blue", 
-      type = "l", 
+# Overlay the malariaEquilibrium Initial EIR to PfPR2-10 line
+lines(x = malEq_P2E$EIR, 
+      y = malEq_P2E$prev, 
+      col = plot_cols[2], 
+      type = "l",
+      lwd = 2,
       lty = 1)
 
-legend("bottomright", legend=c("malariasimulation", "malariaEquilibrium"), col=c("red", "blue"), lty = c(1,1), cex=0.8)
+# Add a legend
+legend("topright",
+       legend = c("malariasimulation", "malariaEquilibrium"),
+       col = c(plot_cols[1:2]), lty = c(1, 1), cex=0.8, box.col = "white")
 
 ```
 
-Now extract values from the fitted line to generate EIR estimates at your desired PfPR2-10 values.
+Looking at the plot, we can see that, to achieve a given *Pf*PR~2-10~, the `malariaEquilibrium` and `malariasimulation` approaches recommend a quite different EIR value, with the `malariaEquilibrium` method typically requiring a much lower EIR value to achieve a given *Pf*PR~2-10~ (given the `malariasimulation` parameter set used).
```

Comment:
Plot looks good! We should also acknowledge here that the `malariasimulation` method included different species and treatment which the `malariaEquilibrium` method did not account for

Reply:
Changed the paragraph after the plot to:

In the example above, we can see that the `malariaEquilibrium` and `malariasimulation` approaches recommend quite different EIR values to achieve a given target *Pf*PR~2-10~, with the `malariaEquilibrium` method typically requiring a much lower EIR value to achieve a given *Pf*PR~2-10~ (given the `malariasimulation` parameter set). These differences are, in part, driven by the fact that in the `malariasimulation` method we simulated a heterogeneous vector population and a clinical treatment regime that treated 45% of the population with the antimalarial artemether lumefantrine (AL).